tests/NgTests/Feed2CatalogTests.cs

```diff
@@ -320,6 +331,18 @@ private Task<HttpResponseMessage> GetCreatedPackages(HttpRequestMessage request)
                     Created = new DateTime(2015, 1, 1),
                     LastEdited = new DateTime(2015, 1, 1),
                     Published = Convert.ToDateTime("1900-01-01T00:00:00Z")
+                },
+                new ODataPackage
+                {
+                    Id = "TestPackage.SemVer2",
+                    Version = "1.0.0-alpha.1+githash",
+                    Description = "A package with SemVer 2.0.0",
+                    Hash = "",
+                    Listed = false,
+
+                    Created = new DateTime(2015, 1, 1),
+                    LastEdited = new DateTime(2015, 1, 1),
```

Comment:
nitpick:

I understand the other `ODataPackage`s do this as well, but I don't think it's possible to have a `Created` and `LastEdited` be the exact same. We should probably test realistic values rather than those that cannot happen.